tasks/5_profile-lookup/profile-lookup.js

```diff
@@ -0,0 +1,40 @@
+export function lookupProfile(firstName, prop){
+    var foundContact = contacts.find(person => person.firstName === firstName);
+    if(foundContact !== undefined){
```

Comment:
Does `foundContact` could have another value like null? What is the purpose of evaluating with undefined exactly?